cmd/kops/create_cluster.go

```diff
@@ -671,6 +671,8 @@ func RunCreateCluster(f *util.Factory, out io.Writer, c *CreateClusterOptions) e
 		cluster.Spec.Networking.Calico = &api.CalicoNetworkingSpec{}
 	case "canal":
 		cluster.Spec.Networking.Canal = &api.CanalNetworkingSpec{}
+	case "kube-router":
+		cluster.Spec.Networking.Kuberouter = &api.KuberouterNetworkingSpec{}
```

Comment:
Sorry if it was not clear from commit message. Kube-router acts as CNI provider for pod-to-pod networking and is a service proxy and network policy enforcer as well. Kube-router is IPVS based service proxy, it replaces Kube-proxy when '--networking=kube-router'. Also it provides a pod-to-pod networking like other CNI providers like weave, Flannel, calico. It also provides network policy implementation.